code/src/index.css

```diff
@@ -11,3 +12,7 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.top-section {
+  display: flex;
+}
```

Comment:
Just wondering about if there is any particular reason to why you style only .top-section here in the css.file? I can see that you have the TopSection that is rendered in App.js, but cannot find any component named like that 🤔